.github/workflows/ci.yml

```diff
@@ -14,15 +14,17 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node_version: [10, 12, 14]
-    name: Test ${{ matrix.os }} node${{ matrix.node_version }}
+        node_version: [10, 14]
```

Comment:
Only testing agains oldest and newest versions now, in order to reduce amount of builds.